api/cluster/resource/templater.go

```diff
@@ -643,6 +643,27 @@ func (t *InferenceServiceTemplater) createAnnotations(modelService *models.Servi
 
 			annotations[knautoscaling.MetricAnnotationKey] = autoscalingMetrics
 			annotations[knautoscaling.TargetAnnotationKey] = targetValue
+
+			// Since the autoscaling target is an INTEGER, users who have set a concurrency target=1 for their
+			// deployments and would like them to scale up even earlier are unable to do so by simply setting the target
+			// as any float < 1. Instead, they would need to tweak the target-utilization-percentage annotation, which
+			// in turn defines the average number of concurrent requests across all existing replicas at which the
+			// Autoscaler will trigger a scale up on the number of replicas.
+			//
+			// Thus, when the user-defined target < 1, we set:
+			//
+			// target utilization = the default target utilization * the user-defined target
+			//
+			// in order to allow the deployment to scale up earlier than when the default target utilization (70) and
+			// user-defined target = 1 is set. Note however, that this does not change the actual 'soft limit' on the
+			// number of requests that each replica can serve, which remains as 1.
+			//
+			// For more details, see: https://knative.dev/docs/serving/autoscaling/concurrency/#target-utilization
+			if modelService.AutoscalingPolicy.MetricsType == autoscaling.Concurrency &&
+				modelService.AutoscalingPolicy.TargetValue < 1 {
+				annotations[knautoscaling.TargetUtilizationPercentageKey] = fmt.Sprintf("%.0f",
+					70*modelService.AutoscalingPolicy.TargetValue)
+			}
```

Comment:
Why don't we just use the value given by the user? I still don't understand the reasoning behind multiplying the `modelService.AutoscalingPolicy.TargetValue` with the default value here (70). It might lead to confusion as to why the value propagated to the isvc is different from the one provided by the user.

Reply:
yea initially we came up with using 1 * the target % to make it below 1. Only after, he found out that float actually works despite the docs 😅

Reply:
Let me reword the description of this PR to make this clearer.

Reply:
Done!